lucet-wasi/tests/guests/fs.c

```diff
@@ -140,19 +224,31 @@ int main(void)
             res++;
         }
     }
-    assert(res == 2003);
+    if (res != 2003) {
+        exit(42);
+    }
 
     res = closedir(dir);
-    assert(res == 0);
+    if (res != 0) {
+        exit(43);
+    }
 
     res = mkdir("/sandbox/a", 0755);
-    assert(res == 0);
+    if (res != 0) {
+        exit(44);
+    }
     res = mkdir("/sandbox/a/b", 0755);
-    assert(res == 0);
+    if (res != 0) {
+        exit(45);
+    }
     res = mkdir("/sandbox/a/b/c", 0755);
-    assert(res == 0);
+    if (res != 0) {
+        exit(46);
+    }
     res = access("/sandbox/a/b/c", R_OK);
-    assert(res == 0);
+    if (res != 0) {
+        exit(47);
```

Comment:
lol I hate this but I love it

Reply:
once you get used to wasmtime having backtraces its real painful to go back